flumine/order/order.py

```diff
@@ -93,6 +93,9 @@ def execution_complete(self) -> None:
         self.date_time_execution_complete = datetime.datetime.utcnow()
         self.update_data.clear()
 
+    def expiring(self) -> None:
```

Comment:
Any reason for `expiring()` rather than `expired()`?

Reply:
Not a good reason. I will change it.